test/src/com/xilinx/rapidwright/design/TestNetTools.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2024, Advanced Micro Devices, Inc.
+ * All rights reserved.
+ *
+ * Author: Wenhao Lin, AMD Research and Advanced Development.
+ *
+ * This file is part of RapidWright.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.xilinx.rapidwright.design;
+
+import java.util.HashSet;
+
+import com.xilinx.rapidwright.support.RapidWrightDCP;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class TestNetTools {
+    /**
+     * Tests the method NetTools.isGlobalClockNet(Net net).
```

Comment:
Don't forget to change all mentions of `isGlobalClockNet` to `isGlobalClock`

Reply:
Did it. Also modified the mentions in the top comment of this PR.